docs/fr/x-sendfile.md

```diff
@@ -0,0 +1,69 @@
+# Servir efficacement les gros fichiers statiques (`X-Sendfile`/`X-Accel-Redirect`)
+
+Habituellement, les fichiers statiques peuvent être servis directement par le serveur web,
+mais parfois, il est nécessaire d'exécuter du code PHP avant de les envoyer :
+contrôle d'accès, statistiques, en-têtes HTTP personnalisés...
+
+Malheureusement, utiliser PHP pour servir de gros fichiers statiques est inefficace comparé à
+à l'utilisation directe du serveur web (surcharge mémoire, diminution des performances...).
+
+FrankenPHP permet de déléguer l'envoi des fichiers statiques au serveur web
+**après** avoir exécuté du code PHP personnalisé.
+
+Pour ce faire, votre application PHP n'a qu'à définir un en-tête HTTP personnalisé
+contenant le chemin du fichier à servir. FrankenPHP se chargera du reste.
+
+Cette fonctionnalité est connue sous le nom de **`X-Sendfile`** pour Apache, et **`X-Accel-Redirect`** pour NGINX.
+
+Dans les exemples suivants, nous supposons que le "document root" du projet est le répertoire `public/`
+et que nous voulons utiliser PHP pour servir des fichiers stockés en dehors du dossier `public/`,
+depuis un répertoire nommé `private-files/`.
+
+## Configuration
+
+Tout d'abord, ajoutez la configuration suivante à votre `Caddyfile` pour activer cette fonctionnalité :
+
+```patch
+    root * public/
+    # ...
+
++	# Needed for Symfony, Laravel and other projects using the Symfony HttpFoundation component
++	request_header X-Sendfile-Type x-accel-redirect
```

Comment:
Without setting `X-Accel-Mapping` using `BinaryFileResponse` Symfony would throw an Exception.

The header is not required when setting the `X-Accel-Redirect` header on a response manually.

Maybe this should be documented too, like, e.g.:


``


```suggestion
+	request_header X-Sendfile-Type x-accel-redirect
+	# Set the following header if you're using BinaryFileResponse (see below for an example)
+	#request_header X-Accel-Mapping /actual/path/to/private-files=/private-files
```

Reply:
Fixed in the last commit and in https://github.com/dunglas/frankenphp-demo/pull/28.

Thanks!